prince-of-versions/src/main/java/co/infinum/princeofversions/UpdateInfo.java

```diff
@@ -0,0 +1,66 @@
+package co.infinum.princeofversions;
+
+import java.util.Map;
+
+import javax.annotation.Nullable;
+
+/**
+ * Represents selected update configuration object based on requirements
+ */
+public class UpdateInfo {
+
+    /**
+     * @param requiredVersion is required_version from selected object
+     */
+    @Nullable
+    private int requiredVersion;
```

Comment:
This is primitive type so it cannot be null. Use Integer to support Nullable types